pegasus/test/test_section_api_helpers.rb

```diff
@@ -485,6 +509,51 @@ class SectionApiHelperTest < SequelTestCase
           assert_equal true, row[:pairing_allowed]
         end
       end
+
+      it 'does not create with alternate script id for user without experiment' do
+        user_id = 15
+        script_id = FakeDashboard::SCRIPT_CSP2_ALT[:id]
+        params = {
+          user: {id: user_id, user_type: 'teacher'},
+          course_id: FakeDashboard::COURSE_CSP[:id],
+
+          # For some reason, params[:script][:id] and params[:script_id] can
+          # both be used to set the script id. However, only
+          # params[:script][:id] is specified by the REST API client, and only
+          # params[:script][:id] is checked for validity. Therefore, only test
+          # params[:script][:id], here and below.
+          #
+          # TODO(dave): remove unused params[:script_id] from the
+          # DashboardSection.create and /v2/sections/.../update APIs.
```

Comment:
This cleanup step should be straight forward to do, but I want to do it in a separate PR so that reverting won't break the script swapping feature.